extensions/mysql/index.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const Promise = require('bluebird');
+const mysql = require('mysql');
+const crypto = require('crypto');
+const omit = require('lodash/omit');
+const cli = require('../../lib');
+
+class MySQLExtension extends cli.Extension {
+    setup(cmd, argv) {
+        // ghost setup --local, skip
+        if (argv.local) {
+            return;
+        }
+
+        cmd.addStage('mysql', this.setupMySQL.bind(this));
+    }
+
+    setupMySQL(argv, ctx, task) {
+        let self = this;
```

Comment:
Just so we keep in line with the ES6 in the rest of the CLI codebase, can we replace all of the `let self = this;` with arrow functions please?